monkey/monkey_island/cc/services/attack/technique_reports/T1087.py

```diff
@@ -0,0 +1,13 @@
+from common.data.post_breach_consts import POST_BREACH_ACCOUNT_DISCOVERY
+from monkey_island.cc.services.attack.technique_reports.pba_technique import \
+    PostBreachTechnique
+
+__author__ = "shreyamalviya"
+
+
+class T1087(PostBreachTechnique):
+    tech_id = "T1087"
+    unscanned_msg = "Monkey didn't try to get a listing of user accounts."
```

Comment:
Logically, no. But I'd rather leave it as it is; it's better to have this message than a blank output in the report in case something goes wrong. WDYT?